Added Listing indexes in postgres guide (#97)
tembo-io/website: docs/postgres_guides/listing-indexes-in-postgres/listing-indexes-in-postgres.md

**Step 1** - Open your terminal and connect it to your desired Postgres database. Follow our guide to setup the connection with the database, [click here](https://tembo.io/docs/postgres_guides/how-to-connect-to-postgres/).

**Step 2** - To list down all the indexes, use this sql query –

```
SELECT indexname AS index_name,
       tablename AS table_name
FROM pg_indexes
WHERE schemaname = 'public';
```

<img src={SqlQuery} width="400" alt="SqlQuery" />

:::note
You can run this same command in the pgAdmin tool and it will return the same results.
:::

Here we have defined `index_name` and `table_name` as alias names, but it’s totally optional. If you want, you can avoid using them.

`index_name` shows the name of the index stored and `table_name` shows the name of the table stored at that particular index.

Here, we have displayed the _indexname_ and _tablename_ from the `pg_indexes`. However, it also retains some additional data too. To know more about it, [check their official documentation](https://www.postgresql.org/docs/current/view-pg-indexes.html).

## Conclusion

In this guide, we discussed different methods to list down all the indexes present in Postgres.

> If you have a desire to expand your knowledge of Postgres, we strongly encourage you to explore our additional [guides](https://tembo.io/docs/category/postgres-guides).

Reviewer: ```suggestion
Want to expand your knowledge of Postgres? Explore our additional [guides](https://tembo.io/docs/category/postgres-guides).
```

> Please make sure to visit carefully crafted [blog posts](https://tembo.io/blog/) to delve into Tembo's extensions.

Reviewer: ```suggestion
Curious about extensions? Check out our [blog posts](https://tembo.io/blog/) to delve into Tembo's extensions.
```
